slack-release-bot-github-action/README.md

```diff
@@ -0,0 +1,200 @@
+# Creating a GitHub Action for your Slack release BOT
+
+Almost all of us love Slack applications and BOTs that automate something. It could be from an alert, so you're notified as soon as a problem happens, or just a random good morning GIF. In this short tutorial I would like to go through the process of creating a GitHub Action that posts a message as soon as a release happens (that is our use case) but the main topic is how you can automate things with GitHub Action, how to reuse "things" across multiple actions and how to start "simple".
+
+At Homeday we use such process to announce new releases from our component library [Homeday Blocks](https://github.com/homeday-de/homeday-blocks) to our `#frontend` Slack channel and also to announce product releases to the relevant squad's channels. The release BOT looks like the one below:
+
+![Release BOT example from Homeday Blocks](./release-bot-example.png)
+
+## Pre-requisite
+
+You need to have a Slack application to perform the next steps. Slack has a great documentation on how to setup a new application: https://api.slack.com/start
+
+The main feature/functionality your application needs to have is: "Incoming Webhooks". It will allow you, later, to create a webhook that will post a message where you want. Each webhook will have an unique URL which you should keep private.
+
+## Creating a GitHub Action to trigger the webhook
+
+GitHub Actions got quite popular in the past 1-2 years and are a good way to setup continuous integration to our applications on GitHub. It has a way more features than just "run tests on every pull request" but I will leave it up to their docs to walk you thorugh the multitude of options it offers: https://docs.github.com/en/actions
+
+In short a GitHub Action has a name, a trigger and job(s). The name helps us to easily identify what the action is supposed to do. The trigger defines when the action will run, for example every push, every pull request, every comment, every release... The jobs are a sequence of steps that the action will perform. Again, GitHub docs will do a better job than me explaning each of these blocks.
```

Comment:
```suggestion
In short, a GitHub Action has a name, a trigger and job(s). The name helps us to easily identify what the action is supposed to do. The trigger defines when the action will run, for example every push, every pull request, every comment, every release... The jobs are a sequence of steps that the action will perform. Again, GitHub docs will do a better job than me explaining each of these blocks.
```

> a GitHub Action has a name, a trigger and job(s)

Are you referring to "Action" or "Workflow" here? Because it's the workflows that have a trigger, actions are just "jobs", and they don't know anything about which event that got triggered.

Reply:
Good call, it causes definitely some confusion. I will revisit this part.